app/controllers/application_controller.rb

```diff
@@ -1,10 +1,157 @@
 require './config/environment'
 
+require 'pry'
 class ApplicationController < Sinatra::Base
 
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "upupdowndownleftrightleftrightabselectstart"
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  get '/signup' do
```

Comment:
This controller is overloaded. Consider separating your concerns by splitting some of these actions into separate controllers (i.e. a users_controller and a tweets_controller).